### ISSUE TITLE
Subhourly HRRR disappeared from sources list

### ISSUE THREAD

**Issue by cloneofghosts** [Pirate-Weather/pirateweather#377]
### Describe the bug

@alexander0042 Just noticed in my HA install that it is showing the HRRR subhourly last update time as unknown. As far as I can tell the data is there in AWS so must be an issue with PW.

Also noticed that the update times for the other models seem to fluctuate sometimes as well. Wonder if its something to do with the different server its on? The dev endpoint is also seeing the same issue.

### Expected behavior

HRRR subhourly data should be integrated.

### Actual behavior

HRRR subhourly data is not integrated.

### API Endpoint

Production

### Location

Ottawa, Ontario

### Troubleshooting steps

- [X] I have searched this repository and [Home Assistant Repository](https://github.com/alexander0042/pirate-weather-ha) to see if the issue has already been reported.
- [X] I have read through the [API documentation](https://pirateweather.net/en/latest/API/) before opening this issue.
- [X] I have not included my API key anywhere in my issue.
- [X] I have written an informative title.

**Comment by cloneofghosts** [comment]
@alexander0042 I also just now noticed that all the models seem to be stuck and not updating?

**Comment by alexander0042** [comment]
Yup! Servers also needed the new auth creds, so they're restarting as well. One is up and running, the other should be there in the next 20 minutes or so

**Comment by cloneofghosts** [comment]
Things seem to be working again but will monitor this for a bit before closing to make sure.

**Comment by alexander0042** [comment]
What a morning! At least ingest is now using the updated scripts, which is something I was going to have to do sooner or later
